README.md

```diff
@@ -8,7 +8,7 @@ installing this chart prior to installing Galaxy helm.
 
 ## Supported software versions
 
-- Kubernetes 1.27+
+- Kubernetes 1.30+
```

Comment:
Does our minimum version change?

Reply:
Not technically but 1.29 is EOL soon so figured I'd update it now: https://endoflife.date/kubernetes